components/cftapps_private_dns/main.tf

```diff
@@ -9,7 +9,7 @@ locals {
     for gateways, gateway in local.gateways : [
       for app in gateway.app_configuration : {
         name   = "${app.product}-${app.component}-${var.env}",
-        ttl    = 300,
+        ttl    = lookup(app, "ttl", 300),
```

Comment:
or should we just update it?

Reply:
As spoke with @timja - currently set via Jenkins to 3600 - will close PR for now and just update to 300. Will look at removing from Jenkins prior to merging Terraform